roadmap.md

```diff
@@ -90,3 +90,10 @@ This scenario loads a web page and prints it as a PDF. In spec terms, this invol
 
 - [x] Some items from the previous milestones
 - [x] [Printing to PDF as Base64-encoded string](https://w3c.github.io/webdriver-bidi/#command-browsingContext-print)
+
+### Incognito mode
+
+This scenario creates a session with browser in incognito mode. In spec terms, this involves:
```

Comment:
Note that "incognito mode" is a Chromism, and the equivalent feature may work slightly differently in different browsers. We need to be clear about which guarantees we are actually intending to offer here.

Reply:
Renamed to "sandbox" mode to avoid vendor-specific naming.

Reply:
I don't think that `Sandbox` mode is good either given that this conflicts with the Sandbox definition of the script module.

Reply:
I'm open for any other name )

Reply:
"Isolated" mode?

Reply:
How about "Extend roadmap: Incognito/Private mode"

(as per https://support.mozilla.org/en-US/kb/private-browsing-use-firefox-without-history, and Safari also uses the "Private" terminology)